src/SFML/System/CMakeLists.txt

```diff
@@ -67,6 +67,12 @@ endif()
 sfml_add_library(System
                  SOURCES ${SRC} ${PLATFORM_SRC})
 
+# enable precompiled headers
+if (SFML_ENABLE_PCH)
+    message(VERBOSE "enabling PCH for SFML library 'sfml-system' (reusable)")
```

Comment:
What is meant with `(reusable)` here? Is there an easy way to clarify the message?